src/beautifiers/rubocop.coffee

```diff
@@ -4,6 +4,8 @@ Requires https://github.com/bbatsov/rubocop
 
 "use strict"
 Beautifier = require('./beautifier')
+path = require('path')
+temp = require('temp').track()
```

Comment:
The base `Beautifier` class has a `tempFile` function to use: https://github.com/Glavin001/atom-beautify/blob/master/src/beautifiers/beautifier.coffee#L106

I want the temporarily file management to be abstracted from the beautifiers in case the implementation needs to change (such as removing `temp` package) it can be done so in the `Beautifier` class and consequently improve all dependent beautifier subclasses.

Reply:
Yes, but that `tempFile` function writes out known content to a temp file and then closes it. In this particular case, Rubocop only needs a temporary file name so it can do the writing itself. I'm not sure how we'd use `tempFile` for that. I can make another function in Beautifier like `tempFileName` that abstracts it. Would that work?

Reply:
Scratch that, we don't need the temp filename at all.